app/loaders/http-client.ts

```diff
@@ -1,11 +1,14 @@
 import axios, { AxiosInstance } from "axios";
+import config from "../config";
 
 export default abstract class HttpClient {
   protected readonly axios: AxiosInstance;
 
   constructor(baseURL: any) {
+    const adtProtocol = config.adt.https ? "https" : "http";
+    const adtBaseUrl = adtProtocol + "://" + config.adt.host;
     this.axios = axios.create({
-      baseURL: baseURL,
+      baseURL: baseURL === "" ? adtBaseUrl : "https://ngx.ampath.or.ke/amrs",
```

Comment:
```suggestion
      baseURL: baseURL === "" ? adtBaseUrl : "amrsUrl",
```
can add the amrsUrl to the config settings